### DISCUSSION TITLE
Does the WebDAV server support SEARCH?

### DISCUSSION THREAD

**Discussion by foucdeg** [pimcore/pimcore#9116] [Q&A / Support] [answered]
# Feature Request

## Feature description

There is a SEARCH method for WebDAV defined in http://www.webdav.org/specs/rfc5323.html (which is an add-on to WebDAV).

I found no documentation saying whether pimcore implements it or not. Does it?

**Comment by brusch** [comment, marked as answer]
Pimcore uses https://sabre.io/dav/ which doesn't support search, as far as I'm aware.

**Reply by foucdeg** [reply]
Thanks for the answer.
